src/environment.js

```diff
@@ -235,8 +281,10 @@ class Environment {
     switch (this.ci) {
       case 'travis':
         return this._env.TRAVIS_PULL_REQUEST !== 'false' ? this._env.TRAVIS_PULL_REQUEST : null;
-      case 'jenkins':
+      case 'jenkins-prb':
```

Comment:
This PR moves our old support for Jenkins + the deprecated pull request builder plugin aside, and calls it 'jenkins-prb'.